drivers/CMakeLists.txt

```
@@ -1,3 +1,13 @@
+set(drivers "builtin;everest;p256-m")
```

Maybe a little easier to maintain and patch?

```suggestion
set(drivers
    builtin
    everest
    p256-m
    )
```